data/defaults.yaml

```diff
@@ -24,6 +24,16 @@ prometheus::pushprox_proxy::package_ensure: 'latest'
 prometheus::pushprox_proxy::package_name: 'pushprox_proxy'
 prometheus::pushprox_proxy::user: 'pushprox_proxy'
 prometheus::pushprox_proxy::version: '0.1.0'
+prometheus::sachet::config_dir: '/etc/sachet'
```

Comment:
instead of addig the data here, can you please direcy add it to the sachet.pp? that enables puppet-strings to lookup the default values and add them to the reference.md

Reply:
Yes of course !

Reply:
Done in https://github.com/voxpupuli/puppet-prometheus/pull/590/commits/b1495da788ea837286b916c851b19bc1008fd4dd